src/button/src/components/Button.js

```diff
@@ -1,5 +1,7 @@
 // @flow
-import React, { useMemo } from 'react';
+import { useMemo } from 'react';
+
+import * as React from 'react';
```

Comment:
I assume this was automatic from the code mod, but wouldn't it make sense to keep the original here?

```
import React, { useMemo } from 'react';
```

Or, alternatively, switch to importing `forwardRef`?

```
import { forwardRef, useMemo } from 'react';
```